zkvm/src/types.rs

```diff
@@ -171,7 +203,7 @@ impl Data {
         }
     }
 
-    /// Encodes blinded Data values.
+    /// Encodes the data item to a byte buffer.
```

Comment:
specify that we encode the opaque version of data?

Reply:
is it better this way?

> /// Encodes the data item to an opaque bytestring.

Reply:
that's great! thanks :)